libs/build_api_v2.py

```diff
@@ -232,17 +233,25 @@ def _build_metric_annotations(
         sources = _sources_from_provenance_and_source_url(field_name, tag_series, log)
 
     anomalies = tag_series.annotations_all_bucket(field_name)
-    anomalies = [
-        AnomalyAnnotation(
-            date=tag.date, original_observation=tag.original_observation, type=tag.tag_type
-        )
-        for tag in anomalies
-    ]
 
-    if not sources and not anomalies:
+    anomaly_tags = []
+    for tag in anomalies:
+        if isinstance(tag, KnownIssue):
+            anomaly = AnomalyAnnotation(
+                date=tag.date, type=tag.tag_type, public_note=tag.public_note
+            )
+        elif isinstance(tag, KnownIssueNoDate):
+            anomaly = AnomalyAnnotation(type=tag.tag_type, public_note=tag.public_note)
```

Comment:
Yeah true, that would definitely be neater--I do think having the speed of the frontend blocks is important/useful for right now, but this would be a more elegant approach